day3/src/actions/index.js

```diff
@@ -0,0 +1,33 @@
+export const CARD_FLIPPED = 'CARD_FLIPPED';
```

Comment:
For oss ja, men vi er jo vant med store prosjekter, gode editorer og mange hotkeys for å finne frem. Vi er også godt vant med begrepene actions, components, containers, reducers, index, import statements osv. Vet ikke hvor mye studentene er vant med? Kanskje jeg undervurderer de.